clients/go/test/integration_test.go

```diff
@@ -245,6 +251,85 @@ func (s *integrationTestSuite) TestActivateJobs() {
 	}
 }
 
+func (s *integrationTestSuite) TestStreamJobs() {
+	// given
+	ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
+	defer cancel()
+
+	deployment, err := s.client.NewDeployResourceCommand().AddResourceFile("testdata/service_task.bpmn").Send(ctx)
+	if err != nil {
+		s.T().Fatal(err)
+	}
+
+	deployedResource := deployment.GetDeployments()[0]
+	s.NotNil(deployedResource)
+
+	process := deployedResource.GetProcess()
+	s.NotNil(process)
+
+	// when
+	// the stream will be closed by the deferred context cancellation
+	ctx, cancel = context.WithCancel(context.Background())
+	defer cancel()
+	jobsChan := make(chan entities.Job)
+	defer close(jobsChan)
+	go s.client.NewStreamJobsCommand().JobType("task").Consumer(jobsChan).Timeout(time.Minute * 5).
+		WorkerName("worker").RequestTimeout(time.Duration(1) * time.Minute).Send(ctx)
+
+	// Await until the stream is created on the broker
+	// TODO: find a less verbose way of doing this
+	streamExists := false
+	for start := time.Now(); !streamExists && time.Since(start) < 5*time.Second; {
+		response, err := http.Get(fmt.Sprintf("http://%s/actuator/jobstreams/remote", s.MonitoringAddress))
```

Comment:
😅

Reply:
Couldn't you just create instances until you receive a job the first time?

Reply:
I was thinking this might hide issues where not all jobs are received or something =/